docs/how-to-guides/azure-deployment.md

```diff
@@ -15,10 +15,35 @@ Feathr has native cloud integration and getting started with Feathr is very stra
 
 1. Get the principal ID of your account by running `az ad signed-in-user show --query id -o tsv` in the link below (Select "Bash" if you are asked to choose one), and write down that value (will be something like `b65ef2e0-42b8-44a7-9b55-abbccddeefff`)
 
-
 [Launch Cloud Shell](https://shell.azure.com/bash)
 
-2. Click the button below to deploy a minimal set of Feathr resources. This is not for production use as we choose a minimal set of resources, but treat it as a template that you can modify for further use. Note that you should have "Owner" access in your subscription to perform some of the actions.
+
+2. To enable authentication on the Feathr UI (which gets created as part of the deployment script) we need to create an Azure Active Directory (AAD) application. Currently it is not possible to create one through ARM template but you can easily create one by running the following CLI commands in the [Cloud Shell](https://shell.azure.com/bash)
+
+```bash
+# This is the prefix you want to name your resources with, make a note of it, you will need it during deployment.
+prefix="YOUR_RESOURCE_PREFIX" 
+
+# Please don't change this name, a corresponding webapp with same name gets created in subsequent steps.
+sitename="${prefix}webapp" 
+
+# This will create the Azure AD application, note that we need to create an AAD app of platform type Single Page Application(SPA). By default passing the redirect-uris with create command creates an app of type web. 
+az ad app create --display-name $sitename --sign-in-audience AzureADMyOrg --web-home-page-url "https://$sitename.azurewebsites.net" --enable-id-token-issuance true
+
+#Fetch the ClientId, TenantId and ObjectId for the created app
+aad_clientId=$(az ad app list --display-name $sitename --query [].appId -o tsv)
+aad_objectId=$(az ad app list --display-name $sitename --query [].id -o tsv)
+aad_tenantId=$(az account tenant list --query [].tenantId -o tsv)
+
+# Updating the SPA app created above, currently there is no CLI support to add redirectUris to a SPA, so we have to patch manually via az rest
+az rest --method PATCH --uri "https://graph.microsoft.com/v1.0/applications/$aad_objectId" --headers "Content-Type=application/json" --body "{spa:{redirectUris:['https://$sitename.azurewebsites.net/.auth/login/aad/callback']}}"
+
+# Make a note of the ClientId and TenantId, you will need it during deployment.
+echo "AAD_CLIENT_ID: $aad_clientId"
+echo "AZURE_TENANT_ID: $aad_tenantId"
+``` 
+
+3. Click the button below to deploy a minimal set of Feathr resources. This is not for production use as we choose a minimal set of resources, but treat it as a template that you can modify for further use. Note that you should have "Owner" access in your subscription to perform some of the actions.
```

Comment:
Actually, user can choose Purview Registry or SQL Registry, and for access control it will also be configurable. Thus, currently deployment seems not `minimal` per my understanding.

Reply:
I did not add that part, but agreed we should change it to be more relevant.

Reply:
Let me address this too in another PR with container registry changes.